### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by cwhatley** [akermu/emacs-libvterm#101]
I'm seeing this today for the first time. The "cmake .." step leaves me with an empty libvterm-prefix folder and then the build fails. Same issue within emacs using vterm-module-compile.

Broke for me at ac68d6b.

**Comment by akermu** [comment]
Can you post the output of `cmake ..` with a clean build folder?

**Comment by cwhatley** [comment]
```
-- The C compiler identification is AppleClang 11.0.0.11000020
-- Check for working C compiler: /Users/cwhatley/Downloads/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Users/cwhatley/Downloads/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done
CMake Error in CMakeLists.txt:
  Imported target "vterm" includes non-existent path

    "/Users/cwhatley/.emacs.d/straight/build/vterm/build/libvterm-prefix/src/libvterm/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "vterm" includes non-existent path

    "/Users/cwhatley/.emacs.d/straight/build/vterm/build/libvterm-prefix/src/libvterm/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "vterm" includes non-existent path

    "/Users/cwhatley/.emacs.d/straight/build/vterm/build/libvterm-prefix/src/libvterm/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by akermu** [comment]
Can you confirm that your issue is fixed with `c209a86`?

**Comment by cwhatley** [comment]
Yes. Thanks!
